perses/samplers/multistate.py

```diff
@@ -6,10 +6,9 @@
 
 from openmmtools.multistate import sams, replicaexchange
 from openmmtools.states import CompoundThermodynamicState, SamplerState, ThermodynamicState
+from perses.dispersed.utils import create_endstates, create_endstates_from_real_systems
```

Comment:
Non-blocker. We can remove `create_endstates` here. Just to make it cleaner.
```suggestion
from perses.dispersed.utils import create_endstates_from_real_systems
```